packages/fxa-auth-server/lib/senders/emails/styles.ts

```diff
@@ -0,0 +1,61 @@
+export const style = `
+.header-text div{
```

Comment:
Oh man, the CSS being string literals feels almost worse than the MJML/EJS lol.

Hmm, I was going to ask if it was possible to adjust these selectors so that they don't often end in a generic element like `div` but looks like MJML is doing its MJML magic here.

Reply:
To any onlookers: we're using template literals here for now because we couldn't figure out how to import partials using EJS and it was blocking. We can switch later.